app/main/views/buyers.py

```diff
@@ -382,6 +384,56 @@ def view_brief_responses(framework_slug, lot_slug, brief_id):
     ), 200
 
 
+@main.route('/frameworks/<framework_slug>/requirements/<lot_slug>/<brief_id>/award', methods=['GET', 'POST'])
+def award_or_cancel_brief(framework_slug, lot_slug, brief_id):
+    form = None
+    errors = {}
+    get_framework_and_lot(
+        framework_slug,
+        lot_slug,
+        data_api_client,
+        allowed_statuses=['live', 'expired'],
+        must_allow_brief=True,
+    )
+    brief = data_api_client.get_brief(brief_id)["briefs"]
+
+    if not is_brief_correct(brief, framework_slug, lot_slug, current_user.id):
+        abort(404)
+
+    if brief['status'] != "closed":
+        abort(404)
```

Comment:
It might not just be this view (the other form views too)

Reply:
I've looked at this as part of https://github.com/alphagov/digitalmarketplace-briefs-frontend/pull/36, but it will need changing once this is merged obviously.